src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/NewUser.vue

```diff
@@ -1,38 +1,78 @@
 <template>
   <PageContainer>
-    <FormContainer>
-      <div class="d-flex flex-column ga-8">
-        <div>
-          <h1>Profile information</h1>
-          <p class="small">The ECE Registry will notify you of important updates regarding your certification</p>
-        </div>
-        <v-form ref="form" validate-on="input">
-          <div class="d-flex flex-column ga-2">
-            <v-text-field v-model="email" label="Email" variant="outlined" color="primary" type="email" :rules="emailRules"></v-text-field>
-            <v-text-field
-              v-model="phoneNumber"
-              label="Phone number"
-              variant="outlined"
-              color="primary"
-              :rules="phoneRules"
-              @keypress="isNumber($event)"
-            ></v-text-field>
-            <v-checkbox v-model="hasAgreed" label="" color="primary" :rules="hasAgreedRules">
-              <template #label>
-                <div>
-                  I have read and accept the
-                  <router-link to="/new-user/terms-of-use">Terms of Use</router-link>
-                </div>
-              </template>
-            </v-checkbox>
-          </div>
-        </v-form>
-        <v-row>
-          <v-btn rounded="lg" color="primary" class="mr-2" @click="submit">Save and continue</v-btn>
-          <v-btn rounded="lg" variant="outlined" @click="logout">Cancel</v-btn>
-        </v-row>
+    <div class="d-flex flex-column ga-8">
+      <div>
+        <h1>Create your My ECE Registry account</h1>
+        <p class="small mt-2">Welcome {{ oidcUserInfo.firstName!.replace(/^(.*?)(\s.*)?$/, "$1") }}.</p>
+      </div>
+      <div>
+        <ECEHeader title="Information from your BC Services Card account" />
+        <p class="small mt-2">We automatically use the following information from your BC Services Card account for your MY ECE Registry account.</p>
+        <p class="small mt-5">Legal name</p>
+        <p class="small mt-2">{{ oidcUserInfo.firstName }} {{ oidcUserInfo.lastName }}</p>
+
+        <p class="small mt-5">Address</p>
+        <p class="small mt-2">{{ oidcUserInfo.address.street_address }}</p>
+        <p class="small">{{ oidcUserInfo.address.locality }}, {{ oidcUserInfo.address.region }} {{ oidcUserInfo.address.postal_code }}</p>
+        <p class="small">{{ oidcUserInfo.address.country }}</p>
       </div>
-    </FormContainer>
+      <v-form ref="form" validate-on="input">
+        <div class="d-flex flex-column ga-2">
+          <ECEHeader title="Contact Information" />
+          <p class="small mt-2">We'll use this to contact you about your account and updates about your application or certificate.</p>
+          <v-text-field
+            v-model="email"
+            class="mt-5 max-width-500"
+            label="Email"
+            variant="outlined"
+            color="primary"
+            type="email"
+            :rules="emailRules"
```

Comment:
We can use the centralized formRules file. 

I think this page was done before we had this in place. 

So for example, this emailRules could be written as 

import * as Rules from "formrules"

[Rules.required('You must enter email whatever"), Rules.email()]

you can take a look how we implemented this here https://github.com/bcgov/ECC-ECER/blob/b4148409b408509130f04fb44fa1d828dfe33323/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceWorkExperienceReferences.vue#L26